mocks/handlers/launchDarkly.ts

```diff
@@ -26,7 +26,7 @@ const getFlags = (toggleFlags: FlagToggles = {}) =>
         flagVersion: 4,
         variation: 1,
         trackEvents: false,
-        value: toggleFlags[flag] || defaultValue,
+        value: toggleFlags[flag] !== undefined ? toggleFlags[flag] : defaultValue,
```

Comment:
Did you mean to do `??` Or can it be `null` sometimes?

Reply:
it can be null sometimes and false, so I wanted to be explicit that it should only use the default if it wasn't explicitly set